libpod/runtime.go

```diff
@@ -406,7 +409,7 @@ func makeRuntime(runtime *Runtime) (err error) {
 	// Setting signaturepolicypath
 	ir.SignaturePolicyPath = runtime.config.SignaturePolicyPath
 	defer func() {
-		if err != nil {
+		if err != nil && store != nil {
```

Comment:
Will the nil check work here? Store is an interface, not a pointer, so you usually can't check against nil directly